test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -16,6 +17,7 @@ import identifyWithLaunchAppLink from '../../../playground/mocks/data/idp/idx/id
 import userIsNotAssignedError from '../../../playground/mocks/data/idp/idx/error-user-is-not-assigned.json';
 
 const BEACON_CLASS = 'mfa-okta-verify';
+const EARLY_CANCEL_CHALLENGE_REQUEST_WAIT_TIME = 4000;
```

Comment:
Do we need to necessarily wait this long?

Reply:
not sure what's the issue with testcafe, but if I set a shorter wait time, the title from first form cannot be asserted correctly.